pvlib/tracking.py

```diff
@@ -69,6 +79,20 @@ class SingleAxisTracker(PVSystem):
     def __init__(self, axis_tilt=0, axis_azimuth=0, max_angle=90,
                  backtrack=True, gcr=2.0/7.0, cross_axis_tilt=0.0, **kwargs):
 
+        arrays = kwargs.get('arrays', [])
+        if len(arrays) > 1:
+            raise ValueError("SingleAxisTracker does not currently support "
+                             "multiple arrays.")
```

Comment:
The more I think about it the more I'm inclined to keep the value error and just remove "currently" from the error message. To me a `NotImplementedError` means that an entire method is not implemented, but I could be convinced otherwise. Furthermore, we may never support multiple arrays in this constructor (#1109).